packages/pages/src/dev/server/middleware/indexPage.ts

```diff
@@ -249,7 +262,11 @@ const createEntityPageListItems = (
            </a>
         </td>
         <td>
-          ${entityId}
+          ${
+            accountId
```

Comment:
Check accountId, universe, and partition. If, for some reason, accountId is defined but the other two aren't you'll still render a link but it won't have an href.

Reply:
checking accountId + universe. if for some reason partition doesn't exist it'll auto go to yext.com